x-pack/plugins/security_solution/public/overview/pages/data_quality.tsx

```diff
@@ -231,41 +255,57 @@ const DataQualityComponent: React.FC = () => {
 
   return (
     <>
-      {indicesExist ? (
+      {indicesExist && isILMAvailable != null ? (
         <SecuritySolutionPageWrapper data-test-subj="ecsDataQualityDashboardPage">
           <HeaderPage subtitle={subtitle} title={i18n.DATA_QUALITY_TITLE}>
-            <EuiToolTip content={INDEX_LIFECYCLE_MANAGEMENT_PHASES}>
-              <FormControlLayout prepend={ilmFormLabel}>
-                <EuiComboBox
-                  id={labelInputId}
-                  data-test-subj="selectIlmPhases"
-                  placeholder={SELECT_ONE_OR_MORE_ILM_PHASES}
-                  renderOption={renderOption}
-                  selectedOptions={selectedOptions}
-                  style={comboBoxStyle}
-                  options={options}
-                  onChange={setSelectedOptions}
+            {isILMAvailable && (
+              <EuiToolTip content={INDEX_LIFECYCLE_MANAGEMENT_PHASES}>
+                <FormControlLayout prepend={ilmFormLabel}>
+                  <EuiComboBox
+                    id={labelInputId}
+                    data-test-subj="selectIlmPhases"
+                    placeholder={SELECT_ONE_OR_MORE_ILM_PHASES}
+                    renderOption={renderOption}
+                    selectedOptions={selectedOptions}
+                    style={comboBoxStyle}
+                    options={options}
+                    onChange={setSelectedOptions}
+                  />
+                </FormControlLayout>
+              </EuiToolTip>
+            )}
+            {!isILMAvailable && startDate && endDate && (
+              <EuiToolTip content={i18n.DATE_PICKER_TOOLTIP}>
+                <EuiSuperDatePicker
+                  start={startDate}
+                  end={endDate}
+                  onTimeChange={onTimeChange}
+                  showUpdateButton={false}
+                  isDisabled={true}
```

Comment:
This ensures (for now) that the date picker always displays `Last 7 days`, but is otherwise disabled, for _serverless_.

In a future update, the date picker may be re-enabled, with consideration of the following changes:
- The `Commonly used` suggestions in the date picker may be filtered to remove suggestions that are greater than 7 days, i.e. `Last 30 days`,  `Last 90 days`, `Last 1 year`
- Display a "no data state" whenever the user selects a range greater than 7 days, with a detailed explanation, similar to the screenshot below, which is displayed in non-serverless deployments when all the `ILM Phases` are unselected:

![no-data-state](https://github.com/elastic/kibana/assets/4459398/0f102d98-c19d-4446-a6ce-3118d0fd5856)
_A "no data state" similar to the one above may be displayed when a range greater than 7 days is selected_

cc @dhru42

Reply:
I'll follow up in this new issue: https://github.com/elastic/kibana/issues/164671